app/plugin_test.go

```diff
@@ -78,36 +80,49 @@ func TestPluginKeyValueStore(t *testing.T) {
 	assert.Equal(t, kv.Value, ret)
 
 	// Test ListKeys
+	assert.Nil(t, th.App.SetPluginKey(pluginId, "key3", []byte("test3")))
```

Comment:
All this stuff looks like a nice place for table testing :)

Reply:
I contemplated this, but the structure of these tests builds upon the previous ones, so exploding into table testing is a little more involved. I'm inclined to defer this for now.